LiveSplit.AutoSplitters.xml

```diff
@@ -13486,10 +13486,10 @@
             <Game>ShadowFlare</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/BurrowBird/Splitters/master/SF_Auto_Splitter.asl</URL>
+            <URL>https://raw.githubusercontent.com/BurrowBird/Splitters/master/SF_Auto_Splitter_v7%202024-03-23.asl</URL>
```

Comment:
Are you sure you want to include version information in the file name? You will have to PR this repo every time you rename the file.

Reply:
My last update was a few years ago, but I guess not.

Reply:
I guess I can leave the link as is and just update the splitter directly, since I can do that instead, right?

Reply:
Sure.

Reply:
I have no idea how to do that though. Cancel this one and create a new request?

Reply:
Just edit the file on your branch again.